package/README.md

```diff
@@ -71,6 +71,17 @@ You can then use that `getClient` function in your server components:
 const { data } = await getClient().query({ query: userQuery });
 ```
 
+#### Multiple clients
+
+If you need to manage multiple clients, for example if you have different endpoints for content previews,
+the function you pass to `registerApolloClient` can accept arbitrary arguments.
+If you're using Typescript, you can hint their types as a tuple:
+
+```ts
+export const { getClient } = registerApolloClient<[string | null]>((previewToken: string | null) => {
+  ...
+});
+
```

Comment:
````suggestion
```ts
export const { getClient } = registerApolloClient<[string | null]>((previewToken: string | null) => {
  ...
});
```
````

It looks like this code block is missing the closing ticks, so the formatting is a bit weird right now 😆. Here is the [rendered README](https://github.com/tremby/apollo-client-nextjs/tree/register-client-allow-args) for reference.

Reply:
As part of this code block, I'd also love to show off _how_ you'd use the argument to ensure people understand that `getClient` is the function affected by the argument. Stealing from the example in the section before this, I'd suggest adding this line:

```ts
const { data } = await getClient(token).query({ query: userQuery });
```